hub/powertoys/text-extractor.md

```diff
@@ -14,9 +14,14 @@ Text Extractor is a convenient way to copy text from anywhere on your screen. Th
 
 With the activation shortcut (default: <kbd>⊞ Win</kbd>+<kbd>Shift</kbd>+<kbd>T</kbd>), you'll see an overlay on the screen. Click and hold your primary mouse button and drag to activate your capture. The text will be saved to your clipboard.
 
+## How to deactivate
+
+Capture mode is deactivated immediately after text in the selected region is recognized and copied to the clipboard.
+You can exit capture mode by pressing <kbd>Esc</kbd> at any moment.
+
 ## Adjust while trying to capture
 
-By holding <kbd>shift</kbd>, you will change from adjusting the capture region's size to moving the capture region. When you release <kbd>shift</kbd>, you will be able to resize again.
+By holding <kbd>Shift</kbd>, you will change from adjusting the capture region's size to moving the capture region. When you release <kbd>Shift</kbd>, you will be able to resize again.
 
 ## Things to note
```

Comment:
You could also place this in a special `Note` box, see https://learn.microsoft.com/contribute/markdown-reference#alerts-note-tip-important-caution-warning

Reply:
I'd prefer to leave it as it is.